chainerx_cc/chainerx/python/cuda/cuda_module.cc

```diff
@@ -0,0 +1,55 @@
+#include "chainerx/python/cuda/cuda_module.h"
+
+#include <cstddef>
+#include <cstdint>
+#include <utility>
+
+#include <pybind11/pybind11.h>
+
+#include "chainerx/backend.h"
+#include "chainerx/context.h"
+#include "chainerx/cuda/cuda_device.h"
+#include "chainerx/cuda/memory_pool.h"
+
+#include "chainerx/python/common.h"
+
+namespace chainerx {
+namespace python {
+namespace cuda {
+namespace cuda_internal {
+
+namespace py = pybind11;
+
+using CudaDevice = chainerx::cuda::CudaDevice;
+
+void* Malloc(void* backend, size_t bytesize, int device_id) {
```

Comment:
Better to avoid `backend` as the parameter name. (because it being a backend is an implementation detail of `Malloc` and `Free`.)

Reply:
What do you think would be a suitable name? It looks to me like a requirement by the caller rather than an implementation detail though.

Reply:
I prefer `param`.
The requirement is to pass the `param` returned by `get_c_allocator()`, not matter what it actually represents.

Reply:
I see, I changed as your suggestion above.